lib/Controller/RoomController.php

```diff
@@ -999,4 +1080,20 @@ public function demoteModerator($token, $participant) {
 
 		return new DataResponse();
 	}
+
+	protected function demoteGuestToModerator(Room $room, string $sessionId): DataResponse {
```

Comment:
Similar issue for this function, demoteGuestFromModerator would be a better name.